### DISCUSSION TITLE
3.8.2  Faders for all instances from single user show to left?

### DISCUSSION THREAD

**Discussion by Burnhaven** [jamulussoftware/jamulus#2414] [General]
I was testing the beta earlier this week and noticed that for each instance I had running, the fader for that instance was kept to the left, but my other instances were spread out amongst other musicians.  Do you plan to group all faders for a single user to the left?
For example when you launch instances for vocal, instrument and backing track.

**Comment by chrisrimple** [comment]
What sort order did you have enabled? Did your Jamulus profile name/alias (in Settings) contain any characters with ASCII values "lower" than "A"? If you have Sort By Name enabled, and your name is something like "0 Chris", then you'll always appear first in the list.

**Reply by Burnhaven** [reply]
I made the changes you suggested, thanks.
